testsuite/objects/__init__.py

```diff
@@ -63,6 +63,33 @@ def to_dict(self):
         return {"ttl": self.ttl, "key": self.key.to_dict()}
 
 
+@dataclass
+class JsonProperty:
```

Comment:
Won't be better to inherit a `Value` class here and add a `name` property instead of creating dataclass? I believe directly creating `JsonProperty` with name and value would be easier than importing both `Value` and `JsonProperty` and creating one dataclass from them. WDYT?

Reply:
Refactored the data classes